config/application.rb

```diff
@@ -6,19 +6,25 @@
 # you've limited to :test, :development, or :production.
 Bundler.require(*Rails.groups)
 
-class Application < Rails::Application
-  # Settings in config/environments/* take precedence over those specified here.
-  # Application configuration should go into files in config/initializers
-  # -- all .rb files in that directory are automatically loaded.
+module Application
```

Comment:
I explicitly remove the application module because the rest of the classes we have are not in the application module...